sync/class.jetpack-sync-module-import.php

```diff
@@ -1,87 +1,97 @@
 <?php
 
+require_once dirname( __FILE__ ) . '/class.jetpack-sync-functions.php';
+
 class Jetpack_Sync_Module_Import extends Jetpack_Sync_Module {
 
-	private $import_end = false;
+	/**
+	 * Tracks which actions have already been synced for the import
+	 * to prevent the same event from being triggered a second time
+	 * @var array
+	 */
+	private $synced_actions = array();
+
+	/**
+	 * A mapping of known importers to friendly names.
+	 * Keys are the class name of the known importer.
+	 * Values are the friendly name.
+	 * @var array
+	 */
+	private static $known_importers = array(
+		'Blogger_Importer' => 'blogger',
+		'LJ_API_Import' => 'livejournal',
+		'MT_Import' => 'mt',
+		'RSS_Import' => 'rss',
+		'WC_Tax_Rate_Importer' => 'woo-tax-rate',
+		'WP_Import' => 'wordpress',
+	);
+
+	/**
+	 * A mapping of action types to sync action name.
+	 * Keys are the name of the import action.
+	 * Values are the resulting sync action.
+	 * @var array
+	 */
+	private static $import_sync_action_map = array(
+		'import_start' => 'jetpack_sync_import_start',
+		'import_done'  => 'jetpack_sync_import_end',
+		'import_end'   => 'jetpack_sync_import_end',
```

Comment:
It may be worth it to comment as to why `import_done` & `import_end` both map to `jetpack_sync_import_end`

Reply:
Thanks for the suggestion. I just pushed a commit that adds a comment explaining why they map to the same sync action.